spec/negotiation.html

```diff
@@ -391,13 +430,19 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-defaultnumberoption" aoid="DefaultNumberOption">
-      <h1>DefaultNumberOption ( _value_, _minimum_, _maximum_, _fallback_ )</h1>
-
-      <p>
-        The abstract operation DefaultNumberOption converts _value_ to a Number value, checks whether it is in the allowed range, and fills in a _fallback_ value if necessary.
-      </p>
-
+    <emu-clause id="sec-defaultnumberoption" type="abstract operation">
+      <h1>
+        DefaultNumberOption (
+          _value_: an ECMAScript language value,
+          _minimum_: an integer,
+          _maximum_: an integer,
```

Comment:
This operation is called with Number and integer values. a85862e will change this.